resources/META-INF/plugin.xml

```diff
@@ -8,10 +8,15 @@
     <vendor email="alex@lonelybytes.com" url="http://github.com/bealex/SwiftLintAppCode">Alexander Babaev</vendor>
 
     <description><![CDATA[
-<p>Provides highlighting of the SwiftLint errors. You can download it here: <a href="https://github.com/realm/SwiftLint">https://github.com/realm/SwiftLint</a>.
-To configure SwiftLint, create ".swiftlint.yml" file in the project root. See <href="https://github.com/realm/SwiftLint#configuration">here</a> for available options.</p>
+    <p>Enables code highlighting of, and auto&ndash;correct for, SwiftLint, a tool to enforce Swift style and
+    conventions.</p>
+    <p>The SwiftLint <pre>reporter</pre> should be configured as <pre>csv</pre> to integrate with this plugin.</p>
```

Comment:
As far as I remember, this is not required, reporter should be configured as a command-line parameter when SwiftLint is run. If it is not the case, this looks like a bug.

Reply:
It may not be, but the usage information indicates that the CSV reporter should be used for the plugin. That's why I added the clarification. See the note attached to step 1 of the installation instructions [in the plugin readme](https://github.com/bealex/SwiftLintAppCode/blob/master/README.md).

 If that's not the case, then there shouldn't be any reference to the CSV reporter at all.

Note also that the SwiftLint plugin offers no mechanism for specifying command line arguments, so it has to be configured in the config file.

Reply:
Let me check this once again.

AFAIR, required arguments must be set in the code and not be dependant on project configuration.

Reply:
Oooh, I see. CSV is hardcoded in the plugin. I think the comment about CSH should be removed from the plugin description altogether. In the original, it sounds like it's a requirement, but that isn't explicitly stated.

I'll update my PR.